catalog-rest-service/src/main/resources/json/schema/entity/data/pipeline.json

```diff
@@ -56,6 +82,29 @@
         }
       },
       "required": ["id", "name"]
+    },
+    "pipelineStatus": {
+      "description": "Series of pipeline executions, its status and task status.",
+      "type": "object",
+      "properties": {
+        "executionDate": {
+          "description": "Date where the job was executed.",
+          "$ref": "../../type/basic.json#/definitions/date"
```

Comment:
Should this be UTC timestamp?

We had some challenges in the past with dateTime conversions. UTC timestamp is a long that can be converted into any desirable format / local timezones without losing precision.

Reply:
This is a good point, we would be missing more recurrent executions as well. Thanks for pointing it out!